### PR TITLE
IPv6 support for govc vm.customize

### LINKED ISSUE #1971: Support IPv6 in govc vm.customize
It would be great to be able to add IPv6 addresses to customization specifications. 

Unfortunately, I do not speak golang (yet). My naive approach to implement IPv6 support `customize.go` failed:

```
for i, ip6 := range cmd.ip6 {
	nic := &spec.NicSettingMap[i]
	nic.Adapter.IpV6Spec.Ip[0] = &types.CustomizationFixedIpV6{IpAddress: ip6, SubnetMask: 64}
}
```

### PR DESCRIPTION
**Pull request by hreese**
This patch adds rudimentary support to configure IPv6 in vm nics (fixes #1971).

### CODE REVIEW

File: govc/vm/customize.go [hunk 1]
```diff
@@ -59,15 +60,11 @@ func (cmd *customize) Register(ctx context.Context, f *flag.FlagSet) {
 	f.StringVar(&cmd.domain, "domain", "", "Domain name")
 	f.StringVar(&cmd.host.Name, "name", "", "Host name")
 	f.Var(&cmd.mac, "mac", "MAC address")
-	cmd.mac = nil
```
Comment (dougm) [inline govc/vm/customize.go:62]: I think these need to stay, there are some apps embedding govc commands that call this func multiple times.
cc @gressd (#1869)

File: govc/vm/customize.go [hunk 2]
```diff
@@ -98,6 +95,36 @@ Examples:
   govc vm.customize -vm VM -tz America/New_York NAME`
 }
 
+// Parse a string of multiple IPv6 addresses with optional netmask separated by space
+func parseIPv6Argument(argv string) (ipconf []types.BaseCustomizationIpV6Generator, err error) {
+	for _, substring := range strings.Split(argv, " ") {
```
Comment (dougm) [inline govc/vm/customize.go:100]: Can we make this comma delimited instead of space?  We use comma elsewhere for similar.

**Review by dougm — COMMENTED** [review]
Thanks @hreese ! A couple of minor requests, otherwise looks good.

**Comment by dougm** [comment]
@hreese @ttuellmann would like to get this merged if you can make the requests changes. Could you also add to the 'Examples' section for the [generated doc](https://github.com/vmware/govmomi/blob/49ef920581d726cd7a23a326e28a89020549c56f/govc/vm/customize.go#L88)?

**Comment by ttuellmann** [comment]
@dougm We'll see to it after the holidays. Apologies for the delay!

**Comment by dougm** [comment]
No worries @ttuellmann, enjoy the holidays.

**Pull request closed without merging**